source/Mission.cpp

```diff
@@ -1277,6 +1358,12 @@ Mission Mission::Instantiate(const PlayerInfo &player, const shared_ptr<Ship> &b
 	subs["<commodity>"] = result.cargo;
 	subs["<tons>"] = to_string(result.cargoSize) + (result.cargoSize == 1 ? " ton" : " tons");
 	subs["<cargo>"] = subs["<tons>"] + " of " + subs["<commodity>"];
+	subs["<outfit>"] = result.OutfitName();
+	subs["<outfit-units>"] = to_string(result.OutfitUnits());
+	subs["<outfit-tons>"] = (to_string(result.OutfitUnitsMassInt())
```

Comment:
```suggestion
	subs["<outfit-tons>"] = to_string(result.OutfitUnitsMassInt()
```
Mb you placed it wrong but that parenthesis does nothing

Reply:
actually you want to remove the one at the beginning and the one at the end of the next line

Reply:
It was a more complex statement at one point, and just didn't clean that up.